moped-editor/src/views/projects/projectView/ProjectComponents/ComponentForm.js

```diff
@@ -0,0 +1,152 @@
+import React from "react";
```

Comment:
This is the form extracted from `ComponentEditModal` that was removed above. It is now rendered in `CreateComponentModal` and `EditComponentModal` which determine what goes in and out of the form and what the submit button does.